README.md

```diff
@@ -1,16 +1,33 @@
 # media-ui-extensions
 Extending the HTMLVideoElement API (`<video>`) to support advanced video player user-interface features
 
-With the addition of [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) to the browser, video player developers can create custom elements that mimic the video tag API, with goals of creating stand-in compatiblity with the video tag and compatibility across players. The video tag API is however lacking some important functions to support modern player UIs, including playback quality/resolution selection and awareness of ads. This repo is intended to capture requests and proposals for those functions.
+With the addition of [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) to the browser, video player developers can create custom elements that mimic the video tag API, with goals of creating stand-in compatibility with the video tag and compatibility across players. The video tag API is however lacking some important functions to support modern player UIs, including playback quality/resolution selection and awareness of ads. This repo is intended to capture requests and proposals for those functions.
 
-## Submitting a proposal
-Use `template.md` to frame your proposal and make a pull-request to add your proposal document to the `proposals` directory.
+## Goals for Proposals
+we want these proposals to be something that we can eventually propose to the Media WG or WhatWG as additional features to the video element and anything related. In the meantime can be used as a spec to keep things interoperable between implementers.
+
+## Before submitting a proposal
+If you have a new idea, it might be worth creating an issue to discuss it a bit first before submitting a proposal to make sure that it's something could be a good fit and won't be less likely to be rejected later on.
```

Comment:
suggestion(blocking): Ambiguous. Per out of band conversation, it appears that every one of our current [Media UI Extensions Issues](https://github.com/video-dev/media-ui-extensions/issues) should have already been proposals, whereas issues should be reserved for much simpler questions like "Would something that models available renditions be a good candidate for a media ui extension proposal? Having some way to to know details about the available renditions in a predictable way would help for building things like a rendition selector, etc. etc.". Maybe consider adding examples and non-examples?

Reply:
I do think that some ambiguity is warranted here.

Like the Rust RFC, the idea is that someone who isn't part of the core team or deeply involved in video doesn't waste time on a detailed proposal if they aren't sure that it could be useful or that there may be enough community buy in. It's definitely a bit easier on the rust side because it's the core team that decides where-as we're trying to make this more of a community thing.

That said, can still be cleared up a bit with some examples.

I do think that until recently, the renditions issue probably couldn't have been a full proposal (or at least would've been in PR for a long time) because there wasn't a good direction for it. The DVR and stream types probably could've been a PR from the beginning.

Reply:
Ah ok... I think I'm still unclear on what in your mind is a good point for something becoming a proposal PR. Maybe that's fine? Taking the renditions proposal as an example, it was pretty clear (imo) early on that **_some_** kind of API would be beneficial as a media-ui-extension, even if there were a lot of details to work through on what it could/would look like. From our followup conversation, I guess I assumed that once we were confident **_something_** should be proposed, it should move from an issue to a PR. We could also solve this by formalizing the process and (to be defined) relevant parties for requesting an issue be "converted" into a draft proposal PR?

Reply:
I think a proposal is something that you can fill out the template pretty well for, as well as include a specific proposal for how to solve the issue at hand. If you have just a specific problem you're trying to solve, or you haven't thought about alternatives or prior art or stuff like that, it probably belongs as an issue first rather than going directly to a PR.

A lot of the proposals we have coming down the line are things that we have been thinking about for a long time and had a pretty good idea for a direction for them. The fact that they did have a lot of discussion doesn't mean it shouldn't have been a PR proposal.

For the renditions issue, we didn't have a clear direction at first, so, opening an issue was a good call. Now, that we have an idea to proceed, making a PR proposal would be the next step to move the renditions proposal forward.

This section definitely applies less to us since we're likely to only submit a proposal that we need. But this also falls into something else you called out re "the team". Like, who is making the final decision for these things? I was on purpose a bit ambiguous, but maybe too ambiguous. For Rust, it's a lot clearer who it is that makes these decisions. Probably should start with the media-chrome committers (and have them have write access to this repo)? Though, at the moment, I think only heff has write access here.